pkgs/sv-bugpoint.nix

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: lowRISC contributors
+# SPDX-License-Identifier: MIT
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  sv-lang,
+  fmt_10,
+  python3,
+}: let
+  rev = "1e078fa961ac5ade9e2506ce63c091b53ad9dbdb";
+in
+  stdenv.mkDerivation {
+    pname = "sv-bugpoint";
+    # take the first 10 characters of the revision
+    version = builtins.head (builtins.match "(.{10}).*" rev);
```

Comment:
I think the latest convention in nixpkgs when a package has no version itself is to use `0-unstable-<DATE>`